tests/res/fm/test_ecl_run_new.py

```diff
@@ -163,6 +163,19 @@ def test_failed_run_OK(self):
         ecl_config = Ecl100Config()
         run(ecl_config, ["SPE1_ERROR", "--version=2019.3", "--ignore-errors"])
 
+    @pytest.mark.equinor_test
+    @tmpdir()
+    def test_no_hdf5_output_with_ecl100(self):
+        self.init_eclrun_config()
+        shutil.copy(
+            os.path.join(self.SOURCE_ROOT, "test-data/local/eclipse/SPE1.DATA"),
+            "SPE1.DATA",
+        )
+        ecl_config = Ecl100Config()
+        run(ecl_config, ["SPE1.DATA", "--version=2019.3"])
+
+        self.assertTrue(os.path.isfile("non-existing-file.HDF5"))
```

Comment:
BTW, think HDF5 uses small letters in the extension 🙂 
```python
>>> import os
>>> os.path.isfile("setup.py")
True
>>> os.path.isfile("setup.PY")
False
```

Reply:
And linux uses a case sensitive filesystem by default while mac doesnt

Reply:
I assumed capital letter as in the rest of eclipse output but I might be wrong, will double check, thanks! However, the tests pass anyway and as you can see in the name of the file I want the test to fail..

Reply:
That is very interesting 🤔